src/scenario.h

```diff
@@ -302,7 +302,7 @@ typedef struct {
 	char custom_strings[0x8000];
 	uint32 game_ticks_1;
 	rct_ride rides[255];
-	uint16 word_01388698;
+	uint16 word_01388698; // TODO: rename to match RCT2_ADDRESS_SAVED_AGE ???
```

Comment:
@duncanspumpkin If I use `(lldb) watchpoint set expression -w read -s 4 -- 0x01388698`, it triggers the watchpoint in `loc_6949B9` in `peep/peep.c` on line 7004.